game/static/game/css/auth.css

```
@@ -34,9 +34,24 @@ body {
     color: var(--text-primary);
     font-family: Outfit, system-ui, sans-serif;
     min-height: 100vh;
+
+    display: flex;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)